build.gradle

```diff
@@ -36,11 +36,13 @@ repositories {
 }
 
 dependencies {
-    compile 'junit:junit:4.12'
+    compile 'org.assertj:assertj-core:3.8.0'
+    compile 'org.opentest4j:opentest4j:1.0.0'

```

Comment:
I don't like dependencies on `assertj` and `opentest4j`, a more straightforward solution would be to depend on `kotlin-test` and use its assertions.

Reply:
I'll try moving to `kotlin-test`. I'm guessing (and hoping it works well on the android space) but I'm unsure.

If something goes wrong, we could move to `assertj-core:1.x` (or using `Java6Assertions` as the entry point which seemingly provides Android support)

Reply:
I like the idea of kotlin-test, since it is built in by kotlin. By deprecating Mocking and building upon kotlin-test we might be able to get platform independent and therefore support all platforms of kotlin (which JetBrains implements the library for).
Maybe that's a nice  goal for a 2.0 release. That way we would also get independent on junit (which this PR aims to do)

Reply:
Exactly, there's no reason why Kluent can't be used with Kotlin JS or Kotlin Native in the future.